diyspk/testdisk/Makefile

```diff
@@ -1,17 +1,16 @@
 SPK_NAME = testdisk
-SPK_VERS = 7.0
-SPK_REV = 2
+SPK_VERS = 7.1
+SPK_REV = 3
```

Comment:
Same remarks for INSTALLER_SCRIPT, SSS_SCRIPT and INSTALL_PREFIX. Please use service-setup.sh